### PR TITLE
Add dgd_fusions_testset

### PR DESCRIPTION
**Pull request by sakshamphul**
Introduction

This is a PR to add dgd fusion test dataset to annoFusedata so that test within annofuse can call dgd data for testing modules

Test : None (no code just a single tsv file and config json file)

### CODE REVIEW

**Review by migbro — APPROVED** [review]
Saksham has explained to me that this repo is linked with the code repo for test purposes. Therefore, only a couple test files will go here, while the code repo will have any related code changes.

**Comment by jharenza** [comment]
Hi- please don't merge - I didn't get a chance to review nor @federicomarini who should review before any merges - can you revert? The main updates I wanted to make are not using "DGD" as information- this is a "custom" file which happens to apply to DGD. This should be flexible to any algorithm. Second, I think we need to fix the quotes in the custom file example- something is wonky.

**Pull request merged**
